OICI-180:lot 14 partie Reporting

Iandraina-etech opened the pull request:
Finalisation de la parte reporting:
-ajout claim act picker
-modification speciality picker
-modif prescriberReport 
-ajout prescriberFosaReport

Iandraina-etech committed:
Add Claim Act Type filter and FOSA report for prescribers

Introduces a new ClaimActTypePicker and related constants, adds a PrescriberFosaReport component with additional filters, and updates PrescriberReport to support multiple authorized health facilities. Also updates translations and integrates new pickers and report routes in the main configuration.

Iandraina-etech committed:
Merge pull request #1 from Iandraina-etech/feature-arkeup

Add Claim Act Type filter and FOSA report for prescribers

vaneck237 commented on src/index.js:
```diff
@@ -73,13 +75,41 @@ const DEFAULT_CONFIG = {
+            .filter(hf => hf?.id) // Filtrer les valeurs nulles
+            .map(hf => decodeId(hf.id));
+        }      
+        console.log(params);
```
il faut retirer ce console log

Iandraina-etech committed:
Require prescriber in claim validation and remove console log

Added a check to ensure 'prescriber' is present in claim validation in ClaimForm. Also removed an unnecessary console.log statement from index.js for cleaner output.
